packages/modules/api-key/src/migrations/Migration20241205122700.ts

```diff
@@ -0,0 +1,32 @@
+import { Migration } from "@mikro-orm/migrations"
+
+export class Migration20241205122700 extends Migration {
+  async up(): Promise<void> {
+    this.addSql(
+      'alter table if exists "api_key" add column if not exists "deleted_at" timestamptz null;'
+    )
+    this.addSql(
+      'alter table if exists "api_key" alter column "type" type text using ("type"::text);'
+    )
+    this.addSql(
+      'alter table if exists "api_key" add constraint "api_key_type_check" check ("type" in (\'publishable\', \'secret\'));'
```

Comment:
question: Are these constraints autogenerated from the enum property in the DML?

Reply:
Yes